tests/Resources/Page/ODH/AiApps/Anaconda.robot

```diff
@@ -0,0 +1,83 @@
+*** Settings ***
```

Comment:
You should think about renaming this to Anaconda.resource

Reply:
you're right I'll do. Thanks

Reply:
Completely agree, the only thing we have to be careful of is renaming all of the imports as well

Reply:
done